From the HEAD of this PR I have

```
(venv) sbesson@Sebastiens-MacBook-Pro-3 ome-model-documentation % make html linkcheck | grep broken
(  ome-tiff/index: line   63) broken    http://www.cellimagelibrary.org/ - HTTPSConnectionPool(host='www.cellimagelibrary.org', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')))
(   ome-tiff/data: line   28) broken    https://eliceirilab.org/software/wiscscan/ - 404 Client Error: Not Found for url: https://eliceirilab.org/software/wiscscan/
(   ome-tiff/code: line   68) broken    https://www.awaresystems.be/imaging/tiff/bigtiff.html - 404 Client Error: Not Found for url: https://www.awaresystems.be/imaging/tiff/bigtiff.html
(  ome-tiff/tools: line   27) broken    https://imagej.nih.gov/ij/ - 404 Client Error: Not Found for url: https://imagej.nih.gov/ij/
(  ome-tiff/index: line   45) broken    https://www.cytivalifesciences.com/ - 403 Client Error: Forbidden for url: https://www.cytivalifesciences.com/
(  ome-tiff/index: line   44) broken    https://www.zeiss.com/microscopy/int/home.html - 404 Client Error: Not Found for url: https://www.zeiss.com/microscopy/int/home.html
make: *** [linkcheck] Error 1
```

Are these builds running anywhere since the documentation has been extracted to this repository @jburel ? Or is the linkcheck expected to be executed as part of the GitHub workflow (currently disabled)?

--- a/conf.py
+++ b/conf.py
@@ -326,5 +326,7 @@ if not (sys.version_info[0] == 2 and sys.version_info[1] <= 5):
 linkcheck_ignore = [
     # See https://www.ncbi.nlm.nih.gov/pmc/about/copyright/
     # The PMC web site restricts access by the default Sphinx agent
-    "https://www.ncbi.nlm.nih.gov/pmc/.*"
+    "https://www.ncbi.nlm.nih.gov/pmc/.*",
+    "https://www.cytivalifesciences.com/",
+    "http://www.cellimagelibrary.org/"
 ]

--- a/ome-tiff/code.rst
+++ b/ome-tiff/code.rst
@@ -65,7 +65,7 @@ with:
 
 .. seealso::
 
-    `BigTIFF file format specification <https://www.awaresystems.be/imaging/tiff/bigtiff.html>`__
+    `BigTIFF file format specification <https://web.archive.org/web/20240706160214/https://www.awaresystems.be/imaging/tiff/bigtiff.html>`__
 
 Modifying a TIFF comment
 ------------------------

--- a/ome-tiff/data.rst
+++ b/ome-tiff/data.rst
@@ -27,7 +27,7 @@ and imaging this sample data.
 
 The datasets were acquired on a multiphoton workstation (2.1 GHz Athlon
 XP 3200+ with 1GB of RAM) using
-`WiscScan <https://eliceirilab.org/software/wiscscan/>`_. All image
+`WiscScan <https://loci.wisc.edu/wiscscan/>`_. All image
 planes were collected at 512 × 512 resolution in 8-bit grayscale, with an
 integration value of 2.
 

--- a/ome-tiff/index.rst
+++ b/ome-tiff/index.rst
@@ -41,7 +41,7 @@ OME-TIFF is supported by:
 
 * `BIOVIA <https://www.3ds.com/products-services/biovia/>`_
 * `Bitplane AG <http://www.bitplane.com/>`_
-* `Carl Zeiss Microscopy GmbH <https://www.zeiss.com/microscopy/int/home.html>`_
+* `Carl Zeiss Microscopy GmbH <https://www.zeiss.com/microscopy/en/home.html>`_
 * `Cytiva <https://www.cytivalifesciences.com/>`_ (formerly GE Healthcare, Applied Precision)
 * `DRVision Technologies LLC <https://www.drvtechnologies.com>`_
 * `Image-Pro by Media Cybernetics, Inc. <https://www.mediacy.com/>`_

--- a/ome-tiff/specification.rst
+++ b/ome-tiff/specification.rst
@@ -14,7 +14,7 @@ An OME-TIFF dataset consists of:
 
 - one or more files in standard TIFF_ format with the file extension
   ``.ome.tif`` or ``.ome.tiff`` or
-  `BigTIFF format <https://www.awaresystems.be/imaging/tiff/bigtiff.html>`_
+  `BigTIFF format <https://web.archive.org/web/20240706160214/https://www.awaresystems.be/imaging/tiff/bigtiff.html>`_
   with one of these same file extensions or a BigTIFF-specific
   extension ``.ome.tf2``, ``.ome.tf8`` or ``.ome.btf``
 - a string of OME-XML metadata embedded in the ImageDescription tag of the
@@ -50,7 +50,7 @@ specification) shows the organization of a TIFF header along with the
 placement of the OME-XML metadata block.  Note this is for the TIFF
 standard specification only; the header structure is slightly
 different for BigTIFF; see the `BigTIFF file format specification
-<https://www.awaresystems.be/imaging/tiff/bigtiff.html>`__. A TIFF file can
+<https://web.archive.org/web/20240706160214/https://www.awaresystems.be/imaging/tiff/bigtiff.html>`__. A TIFF file can
 contain any number of IFDs, with each one specifying an image plane along with
 certain accompanying metadata such as pixel dimensions, physical
 dimensions, bit depth, color table, etc. One of the fields an IFD can

--- a/ome-tiff/tools.rst
+++ b/ome-tiff/tools.rst
@@ -28,7 +28,7 @@ If you are looking for a solution in Java, there are several options.
 Bio-Formats can read OME-TIFF files, as well as convert from many
 third-party formats into OME-TIFF format—see the :doc:`example source code
 page <code>` for specific examples. Alternatively, the open
-source `ImageJ <https://imagej.nih.gov/ij/>`_ application reads
+source `ImageJ <https://imagej.net/ij/>`_ application reads
 multi-page TIFF files, storing the TIFF comment into the associated
 FileInfo object's "description" field.
 

